engines/python/setup/djl_python/rolling_batch/rolling_batch_vllm_utils.py

```diff
@@ -217,16 +217,25 @@ def supports_speculative_decoding() -> bool:
         return False
 
 
-def get_lora_request_params(request: Request, lora_ids: dict) -> dict:
-    result = dict()
-    adapter = request.adapter
-    if adapter is not None:
-        adapter_name = adapter.get_property("name")
-        adapter_path = adapter.get_property("src")
-        adapter_id = lora_ids[adapter_name]
-        result["lora_request"] = LoRARequest(adapter_name, adapter_id,
-                                             adapter_path)
-    return result
+def create_lora_request(lora_name: str, lora_id: int, lora_path: str,
+                        long_lora_max_len: Optional[int],
+                        base_model_name: Optional[str]) -> LoRARequest:
+    params = {
+        "lora_name": lora_name,
+        "lora_int_id": lora_id,
+        "lora_path": lora_path
+    }
+    if long_lora_max_len is not None:
+        params["long_lora_max_len"] = long_lora_max_len
+    if base_model_name is not None:
```

Comment:
Do we need to support this parameter? What happens if base_model_name is different than the current model we currently have hosted in the engine?

Reply:
spoke offline - we'll just remove base_model_name for now since it isn't used by the engine either